specs/WinRT/WinRTAPIContracts.md

```diff
@@ -0,0 +1,78 @@
+
+
+
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust that Microsoft products are sound.  As Windows App SDK evolves and more developers use Windows App SDK in their own projects Windows App SDK needs a way, at build time and run time, to allow apps to light up API features and to gracefully degrade.  This can be achieved by looking at the deltas between APIs.  
+
+In order to allow apps to depend on API versions Windows App SDK will add contracts to all their WinRT APIs.
+
+
+## Architecture Decisions
+1. Contract boundaries
+   1. One contract per feature/component/sub-system
+   2. Contracts will not span across a transport package/repo.
+2. Contract numbering.
+Contracts will be versioned with one number, starting with 1 and is incremented for any interface change.  This version does not correlate with the Windows App SDK version.
```

Comment:
Compare the markdown layout vs the formatted display
![image](https://user-images.githubusercontent.com/72222/156673815-a8fdcc26-7f9a-46df-80c6-fab7cc395188.png)
![image](https://user-images.githubusercontent.com/72222/156673852-e347d61e-a184-431a-91e3-d0b5cb22f9d3.png)

You forgot Markdown happily flows your paragraph instead of treating each newline as significant.

SUGGESTION: Add some minor formatting tweaks e.g.

1. **Contract boundaries**
    1. One contract per feature/component/sub-system
    2. Contracts will not span across a transport package/repo.
2. **Contract numbering** -- Contracts will be versioned...
3. **Enforcement** -- The build systems...

Don't rely on newlines to format your bulleted list